.github/workflows/ci.yml

```diff
@@ -14,24 +14,6 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  build:
-    name: MacOS
-    runs-on: macos-12
```

Comment:
Swift 5.8 is macOS 13-only, and GitHub Actions doesn't support that yet. Given that our main requirement is Linux, though, maybe that's OK?